### PR TITLE
rm: boards: add Kria KV260 board details

### PR DESCRIPTION
**Pull request by igoropaniuk**
Add flashing details for Kria KV260 Vision AI Starter Kit.

Signed-off-by: Igor Opaniuk <igor.opaniuk@foundries.io>

## Readiness

* [ ] Merge (pending reviews)
* [ ] Merge after _date or event_
* [ ] Draft

## Overview

_Why merge this PR? What does it solve?_

## Checklist

_Optional. Add a 'x' to steps taken._
_You can fill this out after opening the PR. "Did I..."_

* [ ] Run spelling and grammar check, preferably with linter.
* [ ] Avoid changing any header associated with a link/reference.
* [ ] Step through instructions (or ask someone to do so).
* [ ] Review for [wordiness](https://languagetool.org/insights/post/wordiness/)
* [ ] Match tone and style of page/section.
* [ ] Run `make linkcheck`.
* [ ] View HTML in a browser to check rendering.
* [ ] Use [semantic newlines](https://bobheadxi.dev/semantic-line-breaks/).
* [ ] follow best practices for commits.
  * [ ] Descriptive title written in the imperative.
  * [ ] Include brief overview of QA steps taken.
  * [ ] Mention any related issues numbers.
  * [ ] End message with sign off/DCO line (`-s, --signoff`).
  * [ ] Sign commit with your gpg key (`-S, --gpg-sign`).
  * [ ] Squash commits if needed.
* [ ] Request PR review by a technical writer and at least one peer.

## Comments

_Any thing else that a maintainer/reviewer should know._
_This could include potential issues, rational for approach, etc._

### CODE REVIEW

File: source/reference-manual/boards/kv260.rst [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+    ## Checking hash(es) for Image optee ... sha256+ OK
+    ## Checking hash(es) for Image fpga ... sha256+ OK
+
+    U-Boot 2022.01+xlnx+g37fad64d18 (Apr 19 2022 - 15:29:09 +0000)
+
+Recovery
+--------
+
+If boot image are eventually corrupted, Kria KV260 Vision AI Starter Kit
+provides Image Recovery application. The Image Recovery application is a
```
Comment (kprosise) [inline source/reference-manual/boards/kv260.rst:81]:
```suggestion
If boot images are eventually corrupted, Kria KV260 Vision AI Starter Kit
provides a Image Recovery application. The Image Recovery application is a
```

File: source/reference-manual/boards/kv260.rst [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+
+|Image of Kria board|
+
+#. Ensure that the power code is disconnected.
+
+#. Connect the Micro-USB cable to the ``J4`` connector for serial console
+   output.
+
+
+Flashing and boot
```
Comment (kprosise) [inline source/reference-manual/boards/kv260.rst:19]:
```suggestion
Flashing and Boot
```

File: source/reference-manual/boards/kv260.rst [hunk 3]
```diff
@@ -0,0 +1,92 @@
...
+-----------------
+
+Flash SD card
+~~~~~~~~~~~~~
```
Comment (kprosise) [inline source/reference-manual/boards/kv260.rst:23]:
```suggestion
Flashing an SD Card
~~~~~~~~~~~~~~~
```

File: source/reference-manual/boards/kv260.rst [hunk 4]
```diff
@@ -0,0 +1,92 @@
...
+.. note:: Device names and IDs can slightly differ from the steps below.
+
+.. include:: generic-flashing.rst
+
+Flash QSPI boot images
+~~~~~~~~~~~~~~~~~~~~~~
+
+The SoM Starter Kits use a two stage boot process:
+
+- the primary boot firmware is pre-installed at the factory on the QSPI device.
```
Comment (kprosise) [inline source/reference-manual/boards/kv260.rst:34]:
```suggestion
- The primary boot firmware is pre-installed at the factory on the QSPI device.
```

File: source/reference-manual/boards/kv260.rst [hunk 5]
```diff
@@ -0,0 +1,92 @@
...
+
+- The secondary boot device is an SD card containing the Linux kernel and
+  system root.
+
+The Xilinx Starter Kit carrier card hardware design sets and factory-locks
+the MPSoC boot mode to QSPI32. The SoM boots up to U-Boot using the QSPI
+contents and U-Boot then hand-off to the secondary boot device.
+To replace boot image on QSPI device with LmP boot images, they have to be
```
Comment (kprosise) [inline source/reference-manual/boards/kv260.rst:42]:
```suggestion
To replace the boot image on QSPI device with LmP boot images, they have to be
```

File: source/reference-manual/boards/kv260.rst [hunk 6]
```diff
@@ -0,0 +1,92 @@
...
+    > sf probe; setenv bootseq 1; mmc dev ${bootseq};
+    > fatload mmc ${bootseq}:1 ${loadaddr} boot0001.bin;
+    > sf update ${loadaddr} 0x200000 ${filesize};
+    > sf update ${loadaddr} 0xF80000 ${filesize};
+    > fatload mmc ${bootseq}:1 ${loadaddr} u-boot0001.itb;
+    > sf update ${loadaddr} 0x300000 ${filesize};
+    > sf update ${loadaddr} 0x1080000 ${filesize};
+    > reset
+
+After reboot LmP U-Boot serial output is shown:
```
Comment (kprosise) [inline source/reference-manual/boards/kv260.rst:56]:
```suggestion
After reboot, LmP U-Boot serial output is shown:
```

**Review by kprosise — APPROVED** [review]
Minor suggestions made, otherwise it LGTM

**Comment by doanac** [comment]
Docs for 95f23472cf0f655a2980f12e3063c3029ce76a5d are browsable at: https://ci.foundries.io/projects/fio-docs/builds/1763/docs/artifacts/html/index.html

**Comment by doanac** [comment]
Docs for 6188291f5ad377ad6feebd0a1b10224e4536a5c2 are browsable at: https://ci.foundries.io/projects/fio-docs/builds/1771/docs/artifacts/html/index.html

**Pull request merged**